DOM_creation/Lesson2/main.js

```diff
@@ -0,0 +1,15 @@
+const ul= document.getElementById("js-ul");
+const li = document.createElement('li');
+
+const a = document.createElement('a');
+a.setAttribute("href", "1.html");
```

Comment:
IMO
If the property is a standard property, it seems preferable to add it with a DOM property rather than using setAttribute. I think the other sections can be changed in the same way.

[reference](https://github.com/kenmori/handsonFrontend/blob/553595c5e3333a76152c0e76ae9c5cc8293b34e6/html/README.md#elemsetattributeid-js-hoge-vs-elemid--js-hoge%E3%81%AE%E4%BD%BF%E3%81%84%E5%88%86%E3%81%91)

```suggestion
a.href = "1.html";
```

Reply:
Thank you Chihiro-san,
I have edited the lines as they are standard property for each element

Reply:
Kindly check my latest commit [8c5a8c2](https://github.com/Saki-Ota/JS_practice/pull/2/commits/8c5a8c26c94e70df408e70620140caee8151c8ab)